src/ray/common/ray_object.cc

```diff
@@ -102,12 +102,18 @@ RayObject::RayObject(rpc::ErrorType error_type, const rpc::RayErrorInfo *ray_err
 }
 
 bool RayObject::IsException(rpc::ErrorType *error_type) const {
-  if (metadata_ == nullptr) {
+  // For performance, assume metadata of >2 chars (e.g., "PYTHON"), is not an error.
+  if (metadata_ == nullptr || metadata_->Size() > 2) {
```

Comment:
I think this optimization makes sense, but we may need some warning or unit test on `ErrorType`

Why don't we add a simple unit test in python that

1. verify 4 == OBJECT_IN_PLASMA
2. verify the total number of enum < 100

Reply:
(or I am okay with just commenting it in common.proto as well)